README.md

```diff
@@ -1 +1,172 @@
-# node-modules-path
\ No newline at end of file
+# Paths, Folders and Modules
+
+## Objectives
+
+1. Work with folders objects and methods such as `__dirname`, `process.cwd()` and `process.env.PWD`
```

Comment:
It's confusing to me that we're coming back to these globals after so many lessons away from them.